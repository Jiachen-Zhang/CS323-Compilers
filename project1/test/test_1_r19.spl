int test_1_r19()
{
  int a = -0x0a;
}
