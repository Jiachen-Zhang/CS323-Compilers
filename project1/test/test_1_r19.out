Error type A at Line 3: unknown lexeme 0x0a
